src/corehost/cli/deps_resolver.cpp

```diff
@@ -164,6 +164,18 @@ void deps_resolver_t::setup_shared_store_probes(
     }
 }
 
+pal::string_t deps_resolver_t::get_probe_directories()
+{
+    pal::string_t directories;
+    for (const auto& pc : m_probes)
```

Comment:
what support cost are we talking about, both the servicing and FX location are actually populated by us ? and any other location is just like a nuget cache location